subprojects/docs/src/docs/release/notes.md

```diff
@@ -264,6 +264,14 @@ More information can be found below:
  
 This was contributed by [Vladimir Sitnikov](https://github.com/vlsi).
 
+### Publication of SHA256 and SHA512 checksums
+
+If you use the `maven-publish` or `ivy-publish` plugins, Gradle will now automatically upload SHA256 and SHA512 signatures, in addition to the traditional but unsecure MD5 and SHA1 signatures.
```

Comment:
Should this be "signatures" or "checkums" here?

Reply:
good point, should be checksums